src/ir_Gree.h

```diff
@@ -76,7 +77,7 @@ union GreeProtocol{
     uint8_t IFeel       :1;
     uint8_t unknown2    :3;  // value = 0b100
     uint8_t WiFi        :1;
-    uint8_t             :1;
+    uint8_t unknown3    :1;  // value = 0b0
```

Comment:
Should `unknown` comments be updated to reflect the remote model, maybe with reference like See `stateReset` or similar?

Reply:
I agree that I think this can be a more useful comment. Perhaps all the comments for unknowns should be updated to something like "Value depends on remote model, see stateReset".

Thinking ahead I could potentially see listing the model names on the comments becoming a tedious list to maintain in a comment, so I don't think we should list the initial values for the respective models in the comment. This would only be an issue if more models were supported that required different values here though,

Reply:
Yes I'm thinking `  // remote model bits, see stateReset` on all of them.